objects_test.go

```diff
@@ -228,17 +228,20 @@ var _ = Describe("Objects", func() {
 			cidr := "25.0.7.0/24"
 			ipAddress := "25.0.7.59/24"
 			mac := "11:22:33:44:55:66"
+			matchClient := "MAC_ADDRESS"
 			fixedAddr := NewFixedAddress(FixedAddress{
 				NetviewName: netviewName,
 				Cidr:        cidr,
 				IPAddress:   ipAddress,
-				Mac:         mac})
+				Mac:         mac,
+				MatchClient: matchClient})
```

Comment:
add a case for incorrect matchClient

Reply:
actually this in in objects_test.go , here we are checking if the objects are done or not.
And we have not written any test cases for Update calls in object_manager_test.go